model/Licensing/Classes/NoneText.md

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
```

Comment:
Proposed to rename NoneText to NoneCopyrightText for consistency within the profile and prevent future naming conflicts.

Reply:
Considering this non-blocking on merge for now.   Please open issue, so we discuss this after merge.